common.py

```diff
@@ -221,13 +221,13 @@ def __init__(self,fname) :
                 self.nbins_gal_shear_read+=nbins
                 if tr.consider_tracer :
                     self.nbins_gal_shear+=nbins
-                if tr.include_m_bias:
```

Comment:
I assume this is intentional? Was this a bug before?

Reply:
Yes, this was a bug when merging with master -- it would cause the number of multiplicative bias parameters to be added to the total number of parameters even when a shear tracer was turned off.